MessageTerminal/MessageTerminal.cxx

```diff
@@ -6,6 +6,7 @@
 #include "Gantry.h"
 
 #include "shared_defs.h"
+#include "TestStandMessages.h"
```

Comment:
snake case? (just recalling from another PR)

Reply:
The "convention" is shared_defs is different than everything else so I need to go change it just haven't got to it yet.
Methods and variables are snake_case, classes are PascalCase, files are PascalCase